### ISSUE TITLE
osmose slave non utilisé

### ISSUE THREAD

**Issue by Marc-marc-marc** [osm-fr/infrastructure#181]
pour mémoire, Il y a quelques slaves qui ne font rien
eurecom
gironde dixit fred : peu de ressource, trop de contraintes
nl-ubuntu
ca-davin03
ca-davin04
freed voulait utiliser l'un de ces 2 derniers pour valider le mode diff/resume

**Comment by vinber** [comment]
on se débarrasse de gironde et je leur signale ?

**Comment by Marc-marc-marc** [comment]
cela me semble nécessaire de leur signaler le problème.
soit ils ont envie d'augmenter la resource et d'en simplifier l'accès
soit on détruit la partie osmose (il y a aussi une autre partie nomée MapContrib, j'ignore ce qu'il en est)

**Comment by vinber** [comment]
c'est moi qui avait tenté de récupérer ces ressources, par convention avec le département, mais je ne suis pas sur que cela soit trés utile aqujourd'hui et le contact avec qui je bossais à l'époque n'est plus vraiment là !
Est-ce que cela peut-être utile ou on jette ?

**Comment by jocelynj** [comment]
@vinber : Oui, on peut leur dire qu'on n'a plus besoin des ressources gironde pour osmose.

**Comment by frodrigo** [comment]
ca-davin03 et ca-davin04, je pense que l'on n'a plus les machines.

Pour eurecom et nl-ubuntu aucune idée.

Pour le mode resume. Comme l'on n'arrive pas à mettre ne place un test, on a commencé à faire ça directement sur la prod (sur une analyse pour une région).

**Comment by frodrigo** [comment]
La backed dédié à l'analyse osmbin n'est plus non plus utilisé, mais la base sert encore à produire les extract. Migration en py3 à faire.
